src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -99,4 +100,17 @@
         public static final int XBOX_PORT = 0;
         public static final double RUMBLE_INTERMISSION_TIME = 0.15;
     }
+
+    public static class IntakeConstants {
+        public static final TrigonTalonSRX MOTOR = IntakeComponents.MOTOR;
+    }
+
+    public static class IntakeOpenerConstants {
+        public static final TrigonTalonSRX MOTOR = IntakeOpenerComponents.MOTOR;
+        public static final double OPERATING_POWER = 0.4;
+        public static double OPENED_ANGLE = 13;
+        public static double CLOSED_ANGLE = 110;
```

Comment:
angle should be relative to the closed angle.

Reply:
please explain more

Reply:
you didn't fix this...